exasol_integration_test_docker_environment/lib/test_environment/analyze_test_container.py

```diff
@@ -92,7 +97,7 @@ def run_task(self):
 
 class DockerTestContainerPush(DockerTestContainerBuildBase, DockerPushParameter):
 
-    def run_task(self):
+    def run_task(self) -> Iterator[DockerCreateImageTask]:
```

Comment:
`DockerCreateImageTask` is not 100% correct, as it can be different classes. Maybe in another PR...